.kotlin_alpha/services/sts/src/main/kotlin/com/kotlin/sts/AssumeRole.kt

```diff
@@ -0,0 +1,89 @@
+// snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+// snippet-sourcedescription:[AssumeRole.kt demonstrates how to obtain a set of temporary security credentials by using AWS Security Token Service (AWS STS).]
+// snippet-keyword:[AWS SDK for Kotlin]
+// snippet-keyword:[AWS Security Token Service (AWS STS)]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[06/07/2021]
+// snippet-sourceauthor:[AWS - scmacdon]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.kotlin.sts
+
+// snippet-start:[sts.kotlin.assume_role.import]
+import aws.sdk.kotlin.services.sts.StsClient
+import aws.sdk.kotlin.services.sts.model.AssumeRoleRequest
+import aws.sdk.kotlin.services.sts.model.StsException
+import kotlin.system.exitProcess
+// snippet-end:[sts.kotlin.assume_role.import]
+
+/**
+ * To make this code example work, create a Role that you want to assume.
+ * Then define a Trust Relationship in the AWS Console. You can use this as an example:
+ *
+ * {
+ *   "Version": "2012-10-17",
+ *   "Statement": [
+ *     {
+ *       "Effect": "Allow",
+ *       "Principal": {
+ *         "AWS": "<Specify the ARN of your IAM user you are using in this code example>"
+ *       },
+ *       "Action": "sts:AssumeRole"
+ *     }
+ *   ]
+ * }
+ *
+ */
+
+suspend fun main(args:Array<String>) {
+
+    val usage = """
+    Usage:
+        <roleArn> <roleSessionName> 
+
+    Where:
+        roleArn - the Amazon Resource Name (ARN) of the role to assume (for example, rn:aws:iam::000008047983:role/s3role). 
```

Comment:
rn -> arn
also, this looks like a real account ID in the ARN. Switch to fictitious ID.

Reply:
fixed